### ISSUE TITLE
Suggested git flow rules

### ISSUE THREAD

**Issue by Eduardoveras** [intellisys/Runaterra-Docs#5]
These are a suggestion, feel free to comment what you think should be improved

1. Only the person that created a pull request should merge it
2. A PR Can only be merge after being reviewd by one person that didnt work in the code
3. A PR should only contain ONE feature or area of work, **DONT MIX UNRELATED THINGS IN ONE PR**
4. All PR must be well formatted (contain the correct tags, name, assigned people, description, etc)
5. After the PR is merged, delete the FEATURE branch
6. Lear how to correctly mix a PR and rebasing

Also, this could be useful and time saving: https://support.gitkraken.com/repositories/git-flow

**Comment by jvinas** [comment]
I agree with these points you mention here. I'll suggest that you identify what format PRs should have and create a pull request in which you edit the file `standards and guidelines.md` .

Also ... check this PR for some changes already proposed #4
